crates/ide_completion/src/render.rs

```diff
@@ -359,16 +367,38 @@ mod tests {
         }
 
         let mut completions = get_all_items(TEST_CONFIG, ra_fixture);
-        completions.sort_by_key(|it| (Reverse(it.relevance()), it.label().to_string()));
+        completions.sort_by_key(|it| {
+            // (Reverse(it.ref_match().map(|m| m.1).unwrap_or(it.relevance())), it.label().to_string())
+            if let Some((mutability, relevance)) = it.ref_match() {
+                let label = format!("&{}{}", mutability.as_keyword_for_ref(), it.label());
+
+                Reverse((relevance, label))
+            } else {
+                Reverse((it.relevance(), it.label().to_string()))
```

Comment:
I don't think we should sort by relevance in this way. `CompletionRelevance` should probably not derive `PartialOrd`, because any sorting should be done by `score`. But really this part of the code shouldn't concern itself with the score, and we don't want the tests here to fail if we change the scoring algorithm. 

In addition it'd be nice not to duplicate the label formatting for the `ref_match`. 

Are you okay with removing this sort?

Reply:
Yep. If sorting can be ignored in test (and it can), sure. I'll remove it with fix of type equality